.github/workflows/ci.yml

```diff
@@ -1,5 +1,5 @@
 name: Specs
-on:
+on:  # yamllint disable-line rule:truthy
```

Comment:
> `# yamllint disable-line rule:truthy`

This was annoying to disable on a case-by-case basis, but the rule looks actually helpful, so I left it enabled and just added inline comments.